package.json

```diff
@@ -50,6 +53,10 @@
     "dependencies": {
         "@apify/http-request": "^1.1.5",
         "@apify/ps-tree": "^1.1.3",
+        "@types/cheerio": "^0.22.15",
+        "@types/node": "^13.1.8",
+        "@types/puppeteer": "^2.0.0",
+        "@types/tough-cookie": "^2.3.6",
```

Comment:
yes, unless you're compiling .ts sources directly in your project. since people will be consuming (JS and TS coders), the generated types hard-depend on those other types

Reply:
Won't do. As @pocesar explained, this is required.